src/support/guide.ts

```diff
@@ -50,6 +52,14 @@ export const ensureGuideExists = async (guidePath: string): Promise<string> => {
     const guideId = await createGuide(guide);
     const parsedGuideId = parseGuideId(guideId);
     console.log(`Guide created: ${parsedGuideId}`);
+    const existingEnvVars = dotenv.config().parsed ?? {};
```

Comment:
I think this change can be removed as well, right?

Reply:
Yup, i'll take that out